crates/bevy_gltf/src/loader.rs

```diff
@@ -280,6 +280,7 @@ fn load_material(material: &Material, load_context: &mut LoadContext) -> Handle<
             albedo: Color::rgba(color[0], color[1], color[2], color[3]),
             albedo_texture: texture_handle,
             unlit: material.unlit(),
+            pbr: Vec2::new(pbr.roughness_factor(), pbr.metallic_factor()),
```

Comment:
I'm inclined to split the `pbr: Vec2<f32>` into separate `roughness_factor: f32` and `metallic_factor: f32`, to make this more clear to the end user.